### ISSUE TITLE
Heads

### ISSUE THREAD

**Issue by ChampOfGamesYT** [Wertzui123/CB-Heads#1]
Werden die Köpfe als Block behandelt also abgebaut?

**Comment by Wertzui123** [comment]
Nein, es sind Entitys. Blöcke sind in Planung, aber ich weiß nicht wie ich es machen soll.

**Comment by ChampOfGamesYT** [comment]
Wie kannst du Issues erlauben obwohl es ein Fork ist?

**Comment by Wertzui123** [comment]
Ich hab sie einfach in den Settings aktiviert

**Comment by Wertzui123** [comment]
I think that this feature won't come in near feature. But let's hope for it!

**Comment by ChampOfGamesYT** [comment]
Okay

**Comment by NxtLeqitYT** [comment]
Habe Head als Block xD Hab nen eigenes

**Comment by Wertzui123** [comment]
Dieser Kommentar ist nur angebracht wenn du es einen von uns geben würdest, also für Private oder öffentliche Zwecke. Wenn du nur angeben willst brauchen wir dich hier nicht. @NxtLeqitYT

**Comment by Wertzui123** [comment]
So,editiert. Wäre nett, musst du aber natürlich nicht :wink:

Aber unterlasse absofort bitte solche Kommentare.

**Comment by note3crafter** [comment]
ich fände es besser den Kopf als block zu machen da als entity bei zu vielen köpfen der server laggt was bei Blöcken ja in diesem sinne nicht der Fall wäre ^^

**Comment by Wertzui123** [comment]
1. Ist dieses Feature zurzeit nicht in Planung 
2. Sind custom blöcke unmöglich, daher währe es Head + Block bzw. nicht weniger lagg
